test/framework/report/report.go

```diff
@@ -0,0 +1,117 @@
+package report
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/ginkgo/v2/types"
+
+	"github.com/kumahq/kuma/pkg/util/files"
+)
+
+var (
+	BaseDir       = "results"
+	DumpOnSuccess = false
+)
+
+func stagingDir() string {
+	return path.Join(BaseDir, "..", "kuma-test-staging")
+}
+
+// AddFileToReportEntry adds a file to the report. The file will be copied to the report directory.
+// It's an alternative to ginkgo.AddReportEntry so that not all logs are kept in memory.
```

Comment:
Does this mean the output directory will also include a file containing the stdout captured on the GitHub Action page?

Reply:
I removed the stdout from GH action as it's too hard to read. So there's simply a file that contains the output (as you can see from the tree above).
You can see the report in the failed test: https://github.com/kumahq/kuma/actions/runs/13593927346/artifacts/2672474508

It's a lot easier to figure what's up as you have a folder with the test name and everything related to it inside